pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -30,51 +30,38 @@ social:
-    .col-50
-      .tutorial-tile
-        %a{href: "/dance"}
-          %img.tutorial-tile-img{src: "/images/dance_party_2019.gif"}
-        .tutorial-info
-          %span
-            %h3.tutorial-info-h= I18n.t(:hoc_overview_dance_title)
-            .smalltext= I18n.t(:hoc_overview_dance_2019_subtitle)
-          %span
-            %a{href: "/dance"}
-              %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
-            .tutorial-info-guide
-              %a{href: "https://curriculum.code.org/hoc/plugged/8/"}
-                = I18n.t(:hoc_overview_view_teacher_guide)
 
-    .col-50
-      .tutorial-tile
-        %a{href: "/oceans"}
-          %img.tutorial-tile-img{src: "/images/fill-485x235/oceans/oceans-overview.png"}
-        .tutorial-info
-          %span
-            %h3.tutorial-info-h=hoc_s(:hoc_overview_oceans_tutorial_header)
-            .smalltext=hoc_s(:hoc_overview_oceans_tutorial_desc)
-          %span
-            %a{href: "/oceans"}
-              %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
-            .tutorial-info-guide
-              %a{href: "https://curriculum.code.org/hoc/plugged/9/"}
-                = I18n.t(:hoc_overview_view_teacher_guide)
 
-- else
-  .full-resource-block{style: "margin-top: 20px; margin-bottom: 20px;"}
-    %a{href: "/dance"}
-      %img.full-resource-image{src: "/images/dance_party_2019.gif"}
-    .tutorial-info{style: "padding: 30px;"}
-      %span
-        %h2.resource-title= I18n.t(:hoc_overview_dance_title)
-        = I18n.t(:hoc_overview_dance_2019_subtitle)
-      %span
-        %a{href: "/dance"}
-          %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
-        .tutorial-info-guide
-          %a{href: "https://curriculum.code.org/hoc/plugged/8/"}
-            = I18n.t(:hoc_overview_view_teacher_guide)
+.tile-container-responsive{style: "margin-top: 20px;"}
+  .col-50
+    .tutorial-tile
+      %a{href: "/dance"}
+        %img.tutorial-tile-img{src: "/images/dance_party_2019.gif"}
+      .tutorial-info
+        %span
+          %h3.tutorial-info-h= I18n.t(:hoc_overview_dance_title)
+          .smalltext= I18n.t(:hoc_overview_dance_2019_subtitle)
+        %span
+          %a{href: "/dance"}
+            %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
+          .tutorial-info-guide
+            %a{href: "https://curriculum.code.org/hoc/plugged/8/"}
+              = I18n.t(:hoc_overview_view_teacher_guide)
+
+  .col-50
+    .tutorial-tile
+      %a{href: "/oceans"}
+        %img.tutorial-tile-img{src: "/images/fill-485x235/oceans/oceans-overview.png"}
+      .tutorial-info
+        %span
+          %h3.tutorial-info-h=hoc_s(:hoc_overview_oceans_tutorial_header)
+          .smalltext=hoc_s(:hoc_overview_oceans_tutorial_desc)
+        %span
+          %a{href: "/oceans"}
+            %button.tutorial-info-start.tutorial-gray= I18n.t(:go)
+          .tutorial-info-guide
+            %a{href: "https://curriculum.code.org/hoc/plugged/9/"}
```

Comment:
Is this translated now too?

Reply:
Good question.  I'm not sure that we translate these curriculum.code.org pages.  It doesn't seem to be translated for Dance Party 2019 either.

Reply:
We can translate curriculum.code.org pages, for example: https://curriculum.code.org/it-it/hoc/plugged/9/ but it looks like these lesson plans are mostly not translated.  Maybe these lesson plan strings are also in the upcoming incoming translations?

Reply:
Apparently so!